src/sentry/static/sentry/app/components/eventOrGroupExtraDetails.jsx

```diff
@@ -49,45 +52,44 @@ const EventOrGroupExtraDetails = createReactClass({
       showAssignee,
       shortId,
     } = this.props;
-    let styles = {};
-    if (subscriptionDetails && subscriptionDetails.reason === 'mentioned') {
-      styles = {color: '#57be8c'};
-    }
 
     return (
       <GroupExtra align="center">
-        {shortId && (
-          <Box mr={2}>
-            <GroupShortId shortId={shortId} />
-          </Box>
-        )}
-        <Flex align="center" mr={2}>
-          {lastSeen && (
-            <React.Fragment>
-              <GroupExtraIcon className="icon icon-clock" />
-              <TimeSince date={lastSeen} suffix={t('ago')} />
-            </React.Fragment>
-          )}
-          {firstSeen &&
-            lastSeen && <span className="hidden-xs hidden-sm">&nbsp;—&nbsp;</span>}
-          {firstSeen && (
-            <TimeSince
-              date={firstSeen}
-              suffix={t('old')}
-              className="hidden-xs hidden-sm"
-            />
-          )}
-        </Flex>
+        {shortId && <GroupShortId shortId={shortId} />}
+        <Times>
+          <div css={overflowEllipsis}>
```

Comment:
For a future PR, it'd be nice if we combined this with `TextOverflow` so we can have a component for this. `TextOverflow` is used in about 10 diff places.